version.txt

```diff
@@ -36,21 +36,21 @@ gdp.json,1.0.2
 gentry_forest_transects.py,1.4.4
 globi_interaction.json,1.0.2
 great_basin_mammal_abundance.json,1.0.1
-harvard_forests.json,1.0.2
+harvard_forests.json,1.0.3
 home_ranges.json,1.1.2
 intertidal_abund_me.py,1.5.3
 iris.json,1.2.1
 la_selva_trees.py,1.4.2
-lakecats_finaltables.json,1.0.0
+lakecats_final_tables.json,1.0.0
 leaf_herbivory.json,1.2.1
-macroalgal_communities.json,1.0.1
+macroalgal_communities.json,1.0.2
 macrocystis_variation.json,1.0.1
 mammal_community_db.json,1.2.1
 mammal_diet.json,1.1.2
 mammal_life_hist.json,1.2.1
 mammal_masses.json,1.3.2
 mammal_metabolic_rate.json,1.2.1
-MammalSuperTree.py,1.2.3
+mammal_super_tree.py,1.2.3
```

Comment:
Update the major version number for scripts  other than `npn`
https://frictionlessdata.io/specs/patterns/#data-package-version

Reply:
@henrykironde, I should update the major versions of only the scripts `harvard-forests` and `macroalgal-communities`. Right?

Reply:
Since we changed the names of all the scripts in the PR, we should update the major version apart from `npn` which only changes to `npn.py, 2.1.4`

```
harvard_forests.json  -> 2.0.0
macroalgal_communities.json  -> 2.0.0
mammal_super_tree.py ->2.0.0
lakecats_final_tables.json ->
```

explanation:
We are changing either the file names or script name in the PR.
If a user was looking for the old file name
`.retriever/scripts/lakecats_finaltables.json`  or 
using the old script name like `macroalgal_communities` instead of `macroalgal_communities`
the tool would fail, since we are breaking backward compatibility due to these changes.

Reply:
@henrykironde, very thanks for the detailed explanation. I have done the suggested changes.